package.json

```diff
@@ -5,13 +5,13 @@
   "main": "build/index.js",
   "scripts": {
     "compile": "tsc",
-    "prepare": "npm run compile && npm run copyFiles",
-    "bg:start": "npm run prepare && pm2 start build/index.js --name telegram-bot",
-    "bg:restart": "npm run prepare && pm2 restart telegram-bot",
+    "postcompile": "cp -r ./src/locales/ ./build/",
+    "bg:start": "npm run compile && pm2 start build/index.js --name telegram-bot",
+    "bg:restart": "npm run compile && pm2 restart telegram-bot",
```

Comment:
Several optional comments
1) [pre/post scripts](https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts) are your friends if you like to do some... pre/post actions :) And if you don't want to figure out new names (`copyFiles` is too general for me)
2) `prepare` is called every time when `npm i`. I dunno is it intentional or not -- just FYI